src/math/pow.rs

```diff
@@ -479,6 +479,7 @@ mod tests {
             .for_each(|s| s.iter().for_each(|val| pow_test(base, *val, expected)));
     }
 
+    #[allow(bare_trait_objects)]
```

Comment:
Could this be updated with `dyn Fn`? Should be fine to allow and avoids a lint attribute!

Reply:
done